reference/product_information/index.rst

```diff
@@ -0,0 +1,93 @@
+
+Many attributes may refer to the same attribute type.
+
+Each attribute type supports some general and special properties.
+
+**Akeneo PIM** provides built in set of attribute types:
+
+================================  ================  ==========================================================================================================
+Code                              Field             Description
+================================  ================  ==========================================================================================================
+pim_catalog_identifier            string            **(!!)** Only one attribute of this type is allowed. Product identification attribute: it may be **sku**, **erp product id** or any other, it may be unque or not unique.
+pim_catalog_boolean               Yes/No            Boolean value: Yes/No i.e. True/False.
+pim_catalog_number                Number            Represents any number value. For example stock quantity.
+pim_catalog_price_collection      Price             Allow to holds collection of numbers according to currency settings.
+pim_catalog_metric                Metric            Extends number and also allows to holds information about metric family and unit.
+pim_catalog_text                  Text              Text field. Holds any text/varchar data.
+pim_catalog_textarea              Text area         The same as text but longer and field may be represented as wysiwyg editor.
+pim_catalog_file                  File              Extends text and linked to media (FileInfoInterface). Allows to upload file of specific type. 
+pim_catalog_image                 Image             The same as file. Allows uploading image/png, image/jpg file types, and is represented as image. 
+pim_catalog_date                  Date              Any date information: expiration date, available on, etc. 
+pim_catalog_simpleselect          Simple-select     Select field that holds single user predefined attribute option.
+pim_catalog_multiselect           Multi-Select      Select field that holds multiple user predefined attribute options.
+================================  ================  ==========================================================================================================
+
+.. note::
+    For information on how to import/export product attributes using CSV format see :doc:`/reference/import_export/formats/attribute`.
+    
+
+Family and Completeness
+-----------------------
+
+The product family represents products prototype (architype, superclass), imposes product information structure and defines the list of required and optional attributes for product that belongs to it. Additional attributes may be added to concrete product as optional.
+
+An attribute may be Global or scoped to specific channel. For example attribute may be required for e-commerce but not for mobile channel.
+
+The product completeness is calculated apart for each channel and locale combination.
```

Comment:
I would add the fact that completeness is calculated from the mandatory attributes of the family. Something like:

> The calculation of the completeness is based on the mandatory attributes of the product family. Each combination channel/locale has its own completeness calculation.